### ISSUE TITLE
Error after 'from pywinauto.application import Application'

### ISSUE THREAD

**Issue by arozehnal** [pywinauto/pywinauto#887]
## Expected Behavior
```
from pywinauto.application import Application
```

## Actual Behavior
```
Traceback (most recent call last):
  File "d:\Devel\python\lib\ctypes\__init__.py", line 121, in WINFUNCTYPE
    return _win_functype_cache[(restype, argtypes, flags)]
KeyError: (<class 'ctypes.HRESULT'>, (<class 'ctypes.c_long'>, <class 'comtypes.automation.tagVARIANT'>, <class 'comtypes.LP_POINTER(IUIAutomationCondition)'>), 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "d:\Devel\python\lib\site-packages\pywinauto\__init__.py", line 89, in <module>
    from . import findwindows
  File "d:\Devel\python\lib\site-packages\pywinauto\findwindows.py", line 42, in <module>
    from . import controls
  File "d:\Devel\python\lib\site-packages\pywinauto\controls\__init__.py", line 36, in <module>
    from . import uiawrapper # register "uia" back-end (at the end of uiawrapper module)
  File "d:\Devel\python\lib\site-packages\pywinauto\controls\uiawrapper.py", line 47, in <module>
    from ..uia_defines import IUIA
  File "d:\Devel\python\lib\site-packages\pywinauto\uia_defines.py", line 181, in <module>
    pattern_ids = _build_pattern_ids_dic()
  File "d:\Devel\python\lib\site-packages\pywinauto\uia_defines.py", line 169, in _build_pattern_ids_dic
    if hasattr(IUIA().ui_automation_client, cls_name):
  File "d:\Devel\python\lib\site-packages\pywinauto\uia_defines.py", line 50, in __call__
    cls._instances[cls] = super(_Singleton, cls).__call__(*args, **kwargs)
  File "d:\Devel\python\lib\site-packages\pywinauto\uia_defines.py", line 60, in __init__
    self.UIA_dll = comtypes.client.GetModule('UIAutomationCore.dll')
  File "d:\Devel\python\lib\site-packages\comtypes\client\_generate.py", line 110, in GetModule
    mod = _CreateWrapper(tlib, pathname)
  File "d:\Devel\python\lib\site-packages\comtypes\client\_generate.py", line 184, in _CreateWrapper
    mod = _my_import(fullname)
  File "d:\Devel\python\lib\site-packages\comtypes\client\_generate.py", line 24, in _my_import
    return __import__(fullname, globals(), locals(), ['DUMMY'])
  File "d:\Devel\python\lib\site-packages\comtypes\gen\_944DE083_8FB8_45CF_BCB7_C477ACB2F897_0_1_0.py", line 1870, in <module>
    ( ['out', 'retval'], POINTER(POINTER(IUIAutomationElement)), 'element' )),
  File "d:\Devel\python\lib\site-packages\comtypes\__init__.py", line 329, in __setattr__
    self._make_methods(value)
  File "d:\Devel\python\lib\site-packages\comtypes\__init__.py", line 698, in _make_methods
    prototype = WINFUNCTYPE(restype, *argtypes)
  File "d:\Devel\python\lib\ctypes\__init__.py", line 123, in WINFUNCTYPE
    class WinFunctionType(_CFuncPtr):
TypeError: item 2 in _argtypes_ passes a union by value, which is unsupported.
```
## Specifications

  - Pywinauto version:  comtypes-1.1.7 pywin32-227 pywinauto-0.6.8 six-1.14.0
  - Python version and bitness: Python 3.7.6, 32bit
  - Platform and OS: Win 10

**Comment by ajujayapal** [comment]
Had the same issue with Python 3.7.6 and 3.8.1. Issue does not occur in 3.7.5 or 3.8.
I think this will be fixed in 3.8.2 ( but not sure yet)

**Comment by arozehnal** [comment]
Yes, Python 3.7.5 does not suffer from this problem.

**Comment by airelil** [comment]
The issue is already tracked in #868. I'm not sure if the updated versions of Python already rolled out, though.

**Comment by vasily-v-ryabov** [comment]
Yes, the fix is already reverted in CPython, but 3.8.2 is not out yet.

**Comment by anterosa** [comment]
With Python 3.7.0 I get same error.
